src/test/java/org/cactoos/io/StickyTest.java

```diff
@@ -82,15 +82,15 @@ void readsRealUrl() throws MalformedURLException {
     }
 
     @Test
-    void readsFileContentSlowlyAndCountsLength() {
+    void readsFileContentSlowlyAndCountsLength() throws Exception {
         final long size = 100_000L;
         new Assertion<>(
             "Must read bytes from a large source slowly and count length",
             new LengthOf(
                 new Sticky(
                     new SlowInput(size)
                 )
-            ).longValue(),
+            ).value(),
             Matchers.equalTo(size)
```

Comment:
@victornoel Could we use `new IsEqual<>(size)` or `new HasValue<>(size)` rather ? Same in other places.